sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced.ipynb

```diff
@@ -109,14 +112,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import pandas as pd\n",
+    "\n",
     "workspace = ml_client.workspaces.get(name=ml_client.workspace_name)\n",
     "\n",
     "output = {}\n",
     "output[\"Workspace\"] = ml_client.workspace_name\n",
     "output[\"Subscription ID\"] = ml_client.connections._subscription_id\n",
     "output[\"Resource Group\"] = workspace.resource_group\n",
     "output[\"Location\"] = workspace.location\n",
-    "output"
+    "pd.set_option(\"display.max_colwidth\", None)\n",
```

Comment:
nit: required only in case of forecasting?

Reply:
Just formatting, made it uniform